content/source/docs/cloud/api/variable-sets.html.md

```diff
@@ -0,0 +1,613 @@
+          {
+            "id": "ws-z6YvbWEYoE168kpq",
+            "type": "workspaces"
+          }
+        ]
+      },
+      "vars": {
+        "data": [
+          {
+            "id": "var-Nh0doz0hzj9hrm34qq"
+            "type": "vars",
+            "attributes": {
+              "key": "c2e4612d993c18e42ef30405ea7d0e9ae",
+              "value": "8676328808c5bf56ac5c8c0def3b7071",
+              "category": "terraform"
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+## List Variable Set
+
+List all variable sets for an organization.
+
+`GET organizations/:organization_name/varsets`
+
+| Parameter            | Description                                                                                                                                                              |
+| -------------------- | ----------------------------------------------------------------------------------------------------- |
+| `:organization_name` | The name of the organization the workspace belongs to.                                                |
+
+List all variable sets for a workspace.
+
+`GET workspaces/:workspace_id/varsets`
+
+| Parameter            | Description      |
+| -------------------- | -----------------|
+| `:workspace_id`      | The workspace ID |
+
+
+### Sample Response
+
+```json
+{
+  "data": [
+    {
+      "id": "varset-mio9UUFyFMjU33S4",
+      "type": "varsets",
+      "attributes":  {
+         "name": "varset-b7af6a77",
+         "description": "Full of vars and such for mass reuse",
+         "is-global": false,
+         "updated-at": "2021-10-29T17:15:56.722Z",
+         "var-count":  5,
+         "workspace-count": 2
+      },
+      "relationships": {
+        "organization": {
+          "data: {"id": "organization_1", "type": "organizations"}
+        },
+        "vars": {
+          "data": [
+           {"id": "var-abcd12345", "type": "vars"},
+           {"id": "var-abcd12346", "type": "vars"},
+           {"id": "var-abcd12347", "type": "vars"},
+           {"id": "var-abcd12348", "type": "vars"},
+           {"id": "var-abcd12349", "type": "vars"}
+          ]
+        },
+        "workspaces": {
+          "data": [
+           {"id": "ws-abcd12345", "type": "workspaces"},
+           {"id": "ws-abcd12346", "type": "workspaces"}
+          ]
+        }
+      }
+    },
+    ...
+  ],
+  "links": {
+    "self": "<page URL>",
+    "first": "<page URL>",
+    "prev": nil,
+    "next": nil,
+    "last": "<page URL>"
+  }
+}
+```
+
+## Relationships
+
+### Variables
+
+## Add Variable
+
+`POST varsets/:varset_external_id/relationships/vars`
+
+| Parameter            | Description                      |
+| -------------------- | ---------------------------------|
+| `:varset_id`         | The variable set ID              |
+
+### Request Body
+
+This POST endpoint requires a JSON object with the following properties as a request payload.
+
+Properties without a default value are required.
```

Comment:
Do we need to do this for every one of these that needs a JSON payload? I only see this language here, but I think we probably need to have it for all of the areas, right?

Reply:
Yeah, I definitely missed this bit in the copy/pasting.